<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app</class>
 <widget class="QWidget" name="app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(119, 119, 179, 150);
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대&quot;;
</string>
   </property>
   <property name="text">
    <string>추억 관리하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대&quot;;</string>
   </property>
   <property name="text">
    <string>추억 찾아보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대B&quot;;</string>
   </property>
   <property name="text">
    <string>자주 찾는 추억</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_file">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 9pt &quot;DX별과그대B&quot;;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>저장파일 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_file">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);

font: 57 9pt &quot;DX별과그대B&quot;;</string>
   </property>
   <property name="text">
    <string>파일 저장하기</string>
   </property>
  </widget>
  <widget class="QWidget" name="file_input_W" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(100, 100, 160, 0);</string>
   </property>
   <widget class="QPushButton" name="file_input_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 9pt &quot;DX별과그대B&quot;;
background-color: rgb(255, 255, 255,150);</string>
    </property>
    <property name="text">
     <string>@</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대&quot;;
background-color: rgb(255, 255, 255,150);
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::TargetMoveAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>추억 이름</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 13pt &quot;DX별과그대B&quot;;
background-color: rgba(100, 100, 160, 0);
border: 1px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;추억을 선택하세요&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>open_file</zorder>
  <zorder>save_file</zorder>
  <zorder>file_input_W</zorder>
  <zorder>label</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>find_contents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>open_fcList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>contents_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_file</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>open_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_file</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>save_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_input_button</sender>
   <signal>clicked()</signal>
   <receiver>app</receiver>
   <slot>save_file_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>app</receiver>
   <slot>display_record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>app</receiver>
   <slot>display_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_fcList()</slot>
  <slot>contents_menu()</slot>
  <slot>find_contents()</slot>
  <slot>open_myfile()</slot>
  <slot>open_files()</slot>
  <slot>save_files()</slot>
  <slot>save_file_input()</slot>
  <slot>display_file()</slot>
  <slot>display_record()</slot>
 </slots>
</ui>
